ortools/xpress/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+file(GLOB _SRCS "*.h" "*.cc")
+set(NAME ${PROJECT_NAME}_xpress)
+
+add_library(${NAME} OBJECT ${_SRCS})
+set_target_properties(${NAME} PROPERTIES
+  CXX_STANDARD 17
+  CXX_STANDARD_REQUIRED ON
+  CXX_EXTENSIONS OFF
+  POSITION_INDEPENDENT_CODE ON
+  )
+target_include_directories(${NAME} PRIVATE
+  ${PROJECT_SOURCE_DIR}
+  ${PROJECT_BINARY_DIR})
+target_link_libraries(${NAME} PRIVATE
+  absl::hash
+  absl::meta
+  absl::memory
+  absl::strings
+  absl::str_format
+  protobuf::libprotobuf
+  ${PROJECT_NAMESPACE}::${PROJECT_NAME}_proto
+  $<$<BOOL:${USE_COINOR}>:Coin::Cbc>)
```

Comment:
I don't think express need/depends on coin-or headers...

Reply:
Oh, thank you !